tests/mocha/login.js

```diff
@@ -0,0 +1,56 @@
+var supertest = require('supertest');
+var should = require('should');
+var lib = require('../lib');
+var user = require('../../models/user');
+
+var ohCloud = supertest.agent('http://localhost:3000');
+
+describe('Test login', function() {
+    before('create necessary user', function (done) {
+        user.register('test', '123', done);
```

Comment:
For whatever reason, done() is not called until the timeout of 2 seconds is reached, but also a timeout of 15 seconds does not work, so I assume, that there's another error which prevents the callback for user.register to be called. However, I was not able to figure out why. If someone has an idea, I'm open for suggestions :)